test/js/testTemplates.js

```diff
@@ -31,24 +33,29 @@ describe('isString filter', () => {
 
 describe('makeNunjucksColors', () => {
   before(function () {
-    this.colors = makeNunjucksColors({
+    this.ctx = {
       herman: {
         displayColors: undefined,
       },
-    });
+      logger: {
+        warn: sinon.fake(),
+      },
+    };
+    this.colors = makeNunjucksColors(this.ctx);
   });
 
   it('exits early on invalid colors', function () {
     const actual = this.colors('not a color');
     assert.strictEqual(actual, null);
+    sinon.assert.calledOnce(this.ctx.logger.warn);
   });
 
   it('switches on formats', function () {
     const actual = this.colors('#fefced');
     const expected = {
       hex: '#fefced',
-      rgb: 'rgb(254, 252, 237)',
-      hsl: 'hsl(53, 89%, 96%)',
+      rgb: 'rgb(99.608% 98.824% 92.941%)',
+      hsl: 'hsl(52.941 89.474% 96.275%)',
```

Comment:
@mirisuzanne What sort of precision do we want here? By default colorjs.io displays 5 significant digits, but we can change that.

Reply:
I think that's probably fine for now. Their picker allows for setting that as a user, which we could consider as a future story. https://colorjs.io/apps/picker/oklch